zk_prover/src/merkle_sum_tree/tests.rs

```diff
@@ -95,39 +95,39 @@ mod test {
         }
     }
 
-    #[test]
-    fn test_mst_with_bigint() {
-        // create new merkle tree with entries that have balances greater than 2^64
-        let merkle_tree =
-            MerkleSumTree::<N_ASSETS>::new("src/merkle_sum_tree/csv/entry_16_bigints.csv").unwrap();
+    // #[test]
```

Comment:
Do you plan to use this test later?

Reply:
Thanks for pointing it out. Removed it in 9b73cd0. 
I don't think this is crucial to maintain as test